sig-security/annual-report-2021.md

```diff
@@ -0,0 +1,81 @@
+    *   Our third-party security audit subproject is moving forward through the complexity of organizing an audit of one of the world’s largest open-source projects. The key challenges are the scale of our codebase and limited auditor experience with the Go language and cloud-native technologies. They are utilizing members’ existing relationships with audit firms, as well as building new ones, to find solutions to these challenges. 
+    *   PodSecurityPolicy was deprecated in the Kubernetes 1.21 release, and SIG Security worked closely with SIG Auth to figure out what comes next. We are delighted to have contributed to [KEP 2579](https://github.com/kubernetes/enhancements/issues/2579) by writing and reviewing proposals, adding our members’ voices to the design meetings, and [blogging about it for the broader Kubernetes community](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/). We hope to make our close collaboration with SIG Auth a model for future cross-SIG efforts throughout Kubernetes. Thanks again to everyone who has been involved!
+*   **Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+    *   We have accepted stewardship of KEPs [1753](https://github.com/kubernetes/enhancements/issues/1753) and [1933](https://github.com/kubernetes/enhancements/issues/1933) from SIG Instrumentation. Their owners and progress remain the same, and we look forward to helping them move toward graduation.
+    *   KEP [1933](https://github.com/kubernetes/enhancements/issues/1933) - Defend against logging secrets via static analysis - [graduated](https://github.com/kubernetes/test-infra/pull/20836) to General Availability! As a SIG, we will continue to provide review and technical input for future static analysis rule updates.
+    *   KEP [2568](https://github.com/kubernetes/enhancements/issues/2568) - Run control-plane as non-root in kubeadm - has been maturing steadily with help from the Kubernetes community. SIG Security has provided a forum to discuss it and feedback to advance it toward implementable status.
+    *   KEP [1981](https://github.com/kubernetes/enhancements/issues/1981) - Support for Windows privileged containers - has matured toward alpha, including SIG Security member feedback. Nice work, team!
+*   **What areas and/or subprojects does the group need the most help with?**
+    *   We always welcome help from anyone aligned with our vision of improving Kubernetes security by learning together, sharing our expertise, and encouraging cross-SIG collaboration. Ideally, we would like to eventually have participating members with crossover to every SIG, providing a natural conduit of ideas and forming a network of helpful hackers throughout Kubernetes.
```

Comment:
Long term a codified liaison program might help. Someone from sig-security that gets involved in another sig they have some expertise in that can be involved in triage sessions or pinged with questions / potentially escalate.